adaptive_navigation/lib/scaffold.dart

```diff
@@ -233,6 +250,7 @@ class AdaptiveNavigationScaffold extends StatelessWidget {
         children: [
           NavigationRail(
             leading: fabInRail ? floatingActionButton : null,
+            minWidth: 56,
```

Comment:
This should stay 72, but position of menu for nav rail state should be shifted to center align

Reply:
The position of the Menu is not controlled by the nav rail but instead by the Drawer. If there is a drawer it will show up. 

The adaptive nav rail has no control on what type of AppBar the user choses and because of this we don't access to the menu button. Instead of modiying the menu button I modified the width of the navigation rail.

Reply:
I think the nav rail default width of 72 should remain.

One option could be to customize the 2 examples so that the menu button is inset more to align with the rail? Not sure if there's a good solution to this out-of-the-box

Reply:
It is OK if this PR closes #3 and not #2. #2 might be more tricky since there may be no simple way to control the inset of the menu without modifying the API

Reply:
Updated PR description so it only closes PR 3